client/src/component/inputText/style.js

```diff
@@ -0,0 +1,9 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  input: {
+    backgroundColor: theme.palette.secondary.main,
+  },
+}));
+
+export default useStyles;
```

Comment:
take a look at customized [text field in the material ui](https://material-ui.com/components/text-fields/#customized-inputs)